doc/scripts/extract_content.py

```diff
@@ -149,6 +149,8 @@ def extract_content(content):
     # modification times unnecessarily, as this triggers documentation
     # rebuilds.
     for output in content.outputs:
+        if 'guides/bluetooth' in output.src:
```

Comment:
What is this about?

Reply:
oops that is test code, I will remove, sorry.